test/systemtest/README.md

```diff
@@ -0,0 +1,44 @@
+# OpenTitan System Tests
+
+System tests are end-to-end tests for the whole OpenTitan system. They operate
+on build outputs (typically in `build-bin`) and can be used to check that the
+full system, consisting of software, hardware, and tooling, works as expected.
+
+Tests can be executed against different target platforms (such as FPGAs or
+simulations), even though not all OpenTitan-supported targets are supported at
+the moment.
+
+## Available tests
+
+* `earlgrey/test_sim_verilator.py`: Run various software tests against a
+  Verilator-built simulation of an Earl Grey design.
+
+## Run the tests
+
+### Prerequisites
+
+Before the tests can be run, the various components of OpenTitan need to be
+built and present in the `BIN_DIR` directory (typically `$REPO_TOP/build-bin`).
+
+Alternatively, the tests can be executed against any pre-built OpenTitan
+snapshot available from GitHub releases, or from CI.
```

Comment:
That's neat. Could you write a few lines about how exactly to do that? (I'm pretty certain I'll have to look it up each time)